src/tests/intg/test_ts_cache.py

```diff
@@ -284,9 +284,10 @@ def test_group_2307bis_update_same_modstamp(ldap_conn,
     sysdb cache update
     """
     ldb_conn = ldb_examine
-    old_sysdb_attrs, old_ts_attrs = prime_cache_group(
-                                                ldb_conn, "group1",
-                                                ("user1", "user11", "user21"))
+    old_sysdb_attrs, old_ts_attrs = prime_cache_group(ldb_conn, "group1",
+                                                      ("user1",
+                                                       "user11",
+                                                       "user21"))
```

Comment:
much nicer will be put `("user1", "user11", "user21")` to some variable and use variable

Reply:
Could you explain why it is not a problem with python-pep8-1.6.2-9?

Reply:
It depends whether E126 is disabled by default or not.